src/containers/TasksContainer.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Created by Katarzyna_Bak on 31.05.2017.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import TasksContainer from './TasksContainer';
+import configureStore from 'redux-mock-store';
+import { addTask, doneTask } from '../actions';
+
+
+describe('[Container] TaskContainer)', () => {
+
+    const initialState = { todos: {
+        present: [
+            {id: 1, name: 'task 1', categoryId: 1, isDone: false},
+            {id: 2, name: 'task 2', categoryId: 1, isDone: false}
+        ]},
+        filter: { type: 'SHOW_ALL', pattern: '' }
+    };
+
+    const initialStateDone = { todos: {
+        present: [
+            {id: 1, name: 'task 1', categoryId: 1, isDone: false},
+            {id: 2, name: 'task 2', categoryId: 1, isDone: false}
+        ]},
+        filter: { type: 'SHOW_DONE', pattern: '' }
+    };
+
+    const mockStore = configureStore();
+    let store, storeDone, taskContainer, taskContainerDone, categoryId;
+
+    beforeEach( () => {
+        categoryId = 1;
+        store = mockStore(initialState);
+        storeDone = mockStore(initialStateDone);
+        taskContainer = shallow(<TasksContainer store={store} categoryId={categoryId} /> );
+        taskContainerDone = shallow(<TasksContainer store={storeDone} categoryId={categoryId} /> );
+    });
+
+    it('render the connected component with filter SHOW_ALL', () => {
+        expect(taskContainer.length).toEqual(1)
+    });
+
+    it('render the connected component with filter SHOW_DONE', () => {
```

Comment:
Checking, if component renders properly should be together with other tests of this component. Not container.